app/views/PillarAnalysis/AnalyticalStatementInput/StoryAnalysisModal/index.tsx

```diff
@@ -521,7 +709,38 @@ function StoryAnalysisModal(props: Props) {
                                 className={styles.editor}
                                 labelContainerClassName={styles.labelContainer}
                                 inputSectionClassName={styles.inputSection}
-                                label="My Analysis"
+                                label={(
+                                    <>
+                                        <div>
+                                            My Analysis
+                                        </div>
+                                        <div className={styles.labelContainerAction}>
+                                            <QuickActionConfirmButton
+                                                name={undefined}
+                                                title="Auto generate my analysis using NLP"
+                                                message="You are about to auto generate my analysis using NLP. This will use the current analytical statement, automatic summary, informations gap, entries and replace the current my analysis. Are you sure you want to continue?"
```

Comment:
```suggestion
                                                message="You are about to auto generate 'My Analysis' using NLP. This will use the current analytical statement, automatic summary, informations gap, entries and replace the current 'My Analysis'. Are you sure you want to continue?"
```